README.md

```diff
@@ -1 +1,56 @@
 # discord-r9k
+
+An experimental [Robot 9000](https://blog.xkcd.com/2008/01/14/robot9000-and-xkcd-signal-attacking-noise-in-chat/) for Discord.
```

Comment:
Maybe add a sentence or two describing the other things the bot currently does/supports.